pom.xml

```diff
@@ -72,6 +89,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
+      <version>1.10.19</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
```

Comment:
This is not referenced anywhere in the code, right? I would guess that [wiremock] (http://wiremock.org/) should have everything that is needed to test this feature.